clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtils.java

```diff
@@ -209,6 +209,15 @@ public String validateString(String name, boolean isRequired) throws ValidateExc
         return value;
     }
 
+    public Boolean validateBoolean(String name, boolean isRequired) {
+        Boolean value = get(name);
+
+        if (value == null && isRequired)
+            throw new ConfigException(name, null, String.format("The OAuth configuration option %s must be non-null", name));
```

Comment:
you have created the detailed error message, so we don't need to pass `name` and `null` to ConfigException

Reply:
Ah! Good catch!

Since I brazenly stole that code from one of the other validation methods, it looks like I should change the other occurrences too.

Reply:
I changed all the occurrences of creating a new `ConfigException` to only use the detailed message (i.e. I removed the first two constructor parameters). Thanks!